apps/src/templates/instructions/utils.js

```diff
@@ -114,8 +114,15 @@ function removeCommentNodes(root) {
  * @param {Boolean} isRtl True if we are displaying in RTL
  */
 export function convertXmlToBlockly(xmlContainer, isRtl) {
-  const xmls = xmlContainer.getElementsByTagName('xml');
+  // Remove any divs and/or spans with class name 'block-space-container'.
```

Comment:
Nit: could you make this comment describe why its here, rather than what it does? I think the code itself describes pretty clearly what it does, but the why is less so.

It took me a minute to realize that the `block-space-container` class is only applied in this file, not a general Blockly thing (is that right?)

Reply:
Nit: might be able to shorten this up to something like:

```
xmlContainer
  .getElementsByClassName('block-space-container)
  .forEach(container => container.remove())
```

Reply:
Agreed. I like the more concise approach of the `forEach` loop here.

Reply:
I decided to try @bencodeorg's suggestion above real quick and found that it didn't _quite_ work because `getElementsByClassName` returns an array-like HTMLCollection value that doesn't have a `forEach` method. Fortunately, it looks like it's pretty easy to convert it to an array with either `Array.from` or the spread operator. https://dev.to/hasnaindev/easiest-way-to-convert-htmlcollection-into-an-array-bkl

I tested that both of these alternatives work. Not sure which is better, but I still think both are more readable than the standard for loop.

```
  Array.from(
    xmlContainer.getElementsByClassName('block-space-container')
  ).forEach(container => container.remove());
```

```
  [...xmlContainer.getElementsByClassName('block-space-container')].forEach(
    container => container.remove()
  );
```

Reply:
hah that is annoying! Should have known there was probably some reason you did it this way Alice :). TIL!

Reply:
Thanks y'all! I think I also prefer the `Array.from` and will update.

Reply:
I also updated the description of the function to include where this util function is used - authored hints and long instructions in the Instructions panel.